pkgs/components/src/icons/index.tsx

```diff
@@ -1,70 +1,71 @@
 import React from "react"
+import account from "@material-ui/icons/Person"
```

Comment:
Does your formatter automatically alphabetize imports? I like it, but we should figure out how to ensure our options are synced up so we don't keep swapping back and forth.